### PR TITLE
Return Windows machines in vbox mode and don't wait on restart SSH commands

### PR DESCRIPTION
**Pull request by wsong**

### CODE REVIEW

**Comment by dhiltgen** [comment]
The pattern I've seen elsewhere in docs for remote powershell scripts is to do something like
```powershell
Restart-Computer; exit
```

Hopefully that'll work, and will be lower impact.

**Comment by wsong** [comment]
Unfortunately, it didn't seem to work for me, even when I did the "encode command in base64" thing that lets you run more complex scripts as a one-liner in powershell.

File: testkit/machines/provisioner.go [hunk 1]
```diff
@@ -161,7 +161,7 @@ func VerifyDockerEngine(m Machine, localCertDir string) error {
 				return
 			}
 
-			ca, cert, key, err := GenerateNodeCerts(localCertDir, m.GetName(), []string{ip, internalIP, m.GetName()})
+			ca, cert, key, err := GenerateNodeCerts(localCertDir, m.GetName(), []string{ip, internalIP, m.GetName(), "127.0.0.1"})
```
Comment (dongluochen) [inline testkit/machines/provisioner.go]: I do not find `GenerateNodeCerts` definition.

Comment (dongluochen) [inline testkit/machines/provisioner.go]: When we add IPv4 address. It may block IPv6 compatibility.

Comment (wsong) [inline testkit/machines/provisioner.go]: It's in the `machines` branch:

https://github.com/docker/docker-e2e/blob/machine/testkit/machines/certs.go#L59

These IP addresses are just used for adding SANs to the certs, so IPv4 vs. IPv6 functionality doesn't matter.

**Comment by wsong** [comment]
This changes are still necessary for Windows to work, so PTAL when you get another chance

File: testkit/machines/provisioner.go [hunk 2]
```diff
@@ -389,12 +389,18 @@ func VerifyDockerEngineWindows(m Machine, localCertDir string) error {
 				return
 			}
 
+			out, err = m.MachineSSH(`powershell mkdir c:\ProgramData\docker\secrets`)
```
Comment (dhiltgen) [inline testkit/machines/provisioner.go]: This feels a little out-of-place in this tree.  I think it should be the responsibility of downstream projects to create directories necessary for their bits and pieces.  (Yes, there's some leakage in here right now with things like firewall ports which I'd like to clean up and get out of this tree, but lets not add new leakage.)

Comment (wsong) [inline testkit/machines/provisioner.go]: Removed.

**Pull request merged**